plugins/plugin-chart-echarts/src/Funnel/controlPanel.tsx

```diff
@@ -0,0 +1,174 @@
+              default: labelType,
+              renderTrigger: true,
+              choices: [
+                [EchartsFunnelLabelTypeType.Key, 'Category Name'],
+                [EchartsFunnelLabelTypeType.Value, 'Value'],
+                [EchartsFunnelLabelTypeType.Percent, 'Percentage'],
+                [EchartsFunnelLabelTypeType.KeyValue, 'Category and Value'],
+                [EchartsFunnelLabelTypeType.KeyPercent, 'Category and Percentage'],
+                [EchartsFunnelLabelTypeType.KeyValuePercent, 'Category, Value and Percentage'],
+              ],
+              description: t('What should be shown on the label?'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'number_format',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Number format'),
+              renderTrigger: true,
+              default: numberFormat,
+              choices: D3_FORMAT_OPTIONS,
+              description: `${t('D3 format syntax: https://github.com/d3/d3-format')} ${t(
+                'Only applies when "Label Type" is set to show values.',
+              )}`,
+            },
+          },
+        ],
+        [
+          {
+            name: 'show_labels',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show Labels'),
+              renderTrigger: true,
+              default: showLabels,
+              description: t('Whether to display the labels.'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'label_line',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Label Line'),
+              default: labelLine,
+              renderTrigger: true,
+              description: t('Draw line from Funnel to label when labels outside?'),
+            },
+          },
+          {
+            name: 'sort',
+            config: {
+              type: 'SelectControl',
+              label: t('sort'),
+              default: sort,
+              renderTrigger: true,
+              choices: [
+                [null, 'Default'],
+                ['ascending', 'Ascending'],
+                ['descending', 'Descending'],
+              ],
+              description: t('Sort data'),
+            },
+          },
+          {
+            name: 'orient',
+            config: {
+              type: 'SelectControl',
+              label: t('orient'),
+              default: orient,
+              renderTrigger: true,
+              choices: [
+                [null, 'Default'],
+                ['vertical', 'Vertical'],
+                ['horizontal', 'Horizontal'],
+              ],
+              description: t('Funnel chart orientation. The options are vertical, horizontal'),
+            },
+          },
+        ],
+        // eslint-disable-next-line react/jsx-key
+        [<h1 className="section-header">{t('Funnel shape')}</h1>],
```

Comment:
This header doesn't have any controls under it. Can we either add the missing controls, or remove the section?

Reply:
rm `section-header`